### ISSUE TITLE
The reason for "all_trajectory = torch.cumsum(naction / 4.0, dim=1)"

### ISSUE THREAD

**Issue by ccx1997** [InternRobotics/NavDP#30]
https://github.com/InternRobotics/NavDP/blob/2afda9f97a971e2aedde7fcec3e652ba332065a6/baselines/navdp/policy_network.py#L112
Hi, thank you for open sourcing this good work!
I am confused on this line. If each point value is relative to its previous point, the cumsum is reasonable. But Why should it be divided by 4?
Moreover, it seems that the input depth map is normalized to 0-1, which loses the absolute metric.

**Comment by wzcai99** [comment]
Hi, the 4.0 is a hyper-parameter for normalization used for training, as the delta action is a relative small value. And the depth is not normalized to 0-1, we feed the absolute metric into the network. This line of [code](https://github.com/InternRobotics/NavDP/blob/2afda9f97a971e2aedde7fcec3e652ba332065a6/baselines/navdp/navdp_server.py#L82) only scale the input depth of uint16 from (0,65536) into (0m,6.5536m).
